React/my-app/src/App.js

```diff
@@ -1,19 +1,78 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import data from './data.json'
-import './App.css';
+import React, { Component } from "react";
+import data from "./data.json";
+import "./App.css";
+import defaultImg from "./photos/default.png";
 
 class App extends Component {
+  state = { books: data, searchType: null };
+
   render() {
     return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
+      <div className="mainContainer">
+        <h1 className="main_title"> wBooks </h1>
+        <form>
+          <select
+            className="selector"
+            name="select"
+            onChange={this.setSearchType}
+          >
+            <option value="null">Seleccionar filtro</option>
+            <option value="title">Titulo</option>
+            <option value="author">Autor</option>
+          </select>
+          {/* <img className="arrow" src="./photos/arrow.png" /> */}
+          <input
+            className="search"
+            type="text"
+            placeholder="Buscar..."
+            name="fname"
```

Comment:
on a similar way with colors, create a strings.js file but place this file at the same level that this index(when you refactored the structure)
So, place all strings taht you need like constants and the import it where you need.